qt/scientific_interfaces/Indirect/FqFitModel.cpp

```diff
@@ -247,42 +247,41 @@ void JumpFitModel::addWorkspace(const std::string &workspaceName) {
                                      FunctionModelSpectra(""));
 }
 
-void JumpFitModel::removeWorkspace(TableDatasetIndex index) {
+void FqFitModel::removeWorkspace(TableDatasetIndex index) {
   m_jumpParameters.erase(getWorkspace(index)->getName());
   IndirectFittingModel::removeWorkspace(index);
 }
 
-JumpFitParameters &
-JumpFitModel::addJumpFitParameters(MatrixWorkspace *workspace,
-                                   const std::string &hwhmName) {
+FqFitParameters &FqFitModel::addFqFitParameters(MatrixWorkspace *workspace,
+                                                const std::string &hwhmName) {
   const auto &foundParameters = m_jumpParameters.find(hwhmName);
   if (foundParameters != m_jumpParameters.end())
     return foundParameters->second;
 
-  const auto parameters = createJumpFitParameters(workspace);
+  const auto parameters = createFqFitParameters(workspace);
   if (parameters.widths.empty() && parameters.eisf.empty())
     throw std::invalid_argument("Workspace contains no Width or EISF spectra.");
   return m_jumpParameters[hwhmName] = std::move(parameters);
 }
 
-std::unordered_map<std::string, JumpFitParameters>::const_iterator
-JumpFitModel::findJumpFitParameters(TableDatasetIndex dataIndex) const {
+std::unordered_map<std::string, FqFitParameters>::const_iterator
+FqFitModel::findFqFitParameters(TableDatasetIndex dataIndex) const {
   const auto ws = getWorkspace(dataIndex);
   if (!ws)
     return m_jumpParameters.end();
```

Comment:
Maybe we could also rename `m_jumpParameters`?

Reply:
Good spot, theres a few things I missed still using jump, I'll sort that.